src/modules/SGEvent.js

```diff
@@ -0,0 +1,15 @@
+import { notifyEventReceived } from './appEvents';
+
+export default {
+  __call: function call(eventName, eventArguments) {
+    console.log(`${`# Received event ${eventName}`}$`);
```

Comment:
I guess you want to leave the console.log for reason (debugging reason in the future)

Reply:
Yes, the same is in other extension we have, this is why I left it.